frontend-ssr/pages/html/index.tsx

```diff
@@ -37,7 +39,7 @@ export default class Html extends React.Component {
           <Link href="/">
             Back to home
           </Link>
-          <MatchCommandClassComponent commands={expressionToMatch} />
+          <MatchCommandClassComponent commands={expressionToMatch} successMessages={matchSuccess}/>
```

Comment:
Instead of having 2 separate props, let's combine them into an Array of Objects, each with two properties. It's easier to manage them this way.